packages/discovery-react-components/src/components/ResultsPagination/ResultsPagination.tsx

```diff
@@ -131,15 +131,13 @@ const ResultsPagination: FC<ResultsPaginationProps> = ({
             totalItems={matchingResults}
             pageSize={actualPageSize}
             pageSizes={pageSizes}
-            // onChange={handleOnChange} // see PageSelector for why this is commented out
+            onChange={handleOnChange}
```

Comment:
These changes taken from https://github.com/watson-developer-cloud/discovery-components/pull/255

Reply:
Welp, unfortunately had to bring back the custom PageSelector. The new `onChange` callback for `unstabled_Pagination` only fires when clicking on the next/prev buttons, but not when setting a new page size. 😢